Edit/clean up (#623)
thegetty/quire-docs: content/about/open-source.md

online: false
---

**As of December 2022 Quire is officially open source. View our [3-clause BSD open source license](https://github.com/thegetty/quire/blob/main/LICENSE).**

Open source is a way of developing software where the code is free and available for others to use and adapt. Open-source software relies on both transparency and an engaged community of users. The community works together to both use and contribute to the development of the software and its code.

For the past four years, Getty has been using Quire to advance its internal publishing needs. This work has enabled us to test and refine the software. By open-sourcing Quire and supporting its ongoing use in the field, other institutions can leverage Getty's work to create their own digital publications and meaningfully contribute to Quire's growth and health as a publishing tool.

> Quire users have full access and can create, customize, and distribute their digital projects any way they'd like. We encourage users to share questions, frustrations, and positive reactions, and we use this feedback to inform developments and new feature considerations. In return, our users not only have access to Getty staff for assistance, but they also have access to other beta users who can provide additional support and inspiration. The one thing we ask is that users do NOT redistribute Quire itself.

Reviewer: We need to remove the mention of beta, and that we don't allow redistribution. Like:

```suggestion
Quire users have full access and can create, customize, and distribute their digital projects any way they'd like. We encourage users to share questions, frustrations, and positive reactions in our [community forum](https://github.com/thegetty/quire/discussions), and we use this feedback to inform developments and new feature considerations.
```
